src/common/Rates/styled.js

```diff
@@ -96,9 +96,9 @@ export const TotalScore = styled.span`
 `;
 
 export const Votes = styled.p`
-  color: ${({ theme }) => theme.colors.black};
+  color: ${({ theme }) => theme.colors.waterloo};
   margin: 0;
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
I would not change none of this styles in this file, they already matches the design.
Default styles of all styled-components in this file, refers to components like Tile, or DetailsTile where the sizes of elements are smaller. If some of the styled-components gets $larger attribude, its increase their sizes or change layout to match MovieBanner component design, where all styles in Rates component are larger.

Reply:
the project Tile component on MovieListPage scores and votes have the same font size (on large screens 16px, on small 13px). Reducing the font on one of them is visible and the effect is different from the project.
maybe let's create a separate styled component for tile in movie details?